### PR TITLE
simplify Gatekeeper config to exclude Gatekeeper

### PR DESCRIPTION
**Pull request by ch-stark**

### CODE REVIEW

File: community/CM-Configuration-Management/policy-gatekeeper-config-exclude-namespaces.yaml [hunk 1]
```diff
@@ -33,85 +33,12 @@ spec:
...
-                        - openshift-ovirt-infra
-                        - openshift-ovn-kubernetes
-                        - openshift-sdn
-                        - openshift-service-ca
-                        - openshift-service-ca-operator
-                        - openshift-storage
-                        - openshift-user-workload-monitoring
-                        - openshift-vsphere-infra
+                        - open-cluster-management-*
+                        - openshift-*
```
Comment (dhaiducek) [inline community/CM-Configuration-Management/policy-gatekeeper-config-exclude-namespaces.yaml:41]: You still need `openshift` in this case
```suggestion
                        - openshift
                        - openshift-*
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thank goodness for wildcards! 😄

**Comment by ch-stark** [comment]
@dhaiducek why do I need openshift?  I believe there is no openshift namespace,  thanks for the review

**Comment by dhaiducek** [comment]
> @dhaiducek why do I need openshift? I believe there is no openshift namespace, thanks for the review

It was in the original list, so I was basing my review on that.

**Comment by ch-stark** [comment]
this is an example for Gatekeeper-Community:

```
apiVersion: config.gatekeeper.sh/v1alpha1
kind: Config
metadata:
  name: config
  namespace: gatekeeper-system
spec:
  match:
  - excludedNamespaces:
    - kube-*
    - openshift-*
    processes:
    - audit
    - webhook
    - sync
```

**Review by dhaiducek — APPROVED** [review]
Since the `openshift` namespace doesn't actually exist, this LGTM!

File: community/CM-Configuration-Management/policy-gatekeeper-config-exclude-namespaces.yaml [hunk 2]
```diff
@@ -33,85 +33,12 @@ spec:
...
+                        - kube-*
                         - local-quay
                         - multicluster-engine
                         - open-cluster-management
-                        - open-cluster-management-addon-observability
-                        - open-cluster-management-agent
-                        - open-cluster-management-agent-addon
-                        - open-cluster-management-hub
-                        - open-cluster-management-observability
-                        - openshift
```
Comment (thomasmckay) [inline community/CM-Configuration-Management/policy-gatekeeper-config-exclude-namespaces.yaml:48]: keep this one

**Pull request merged**